desc/objectives/_geometry.py

```diff
@@ -297,8 +299,9 @@ class Volume(_Objective):
 
     Parameters
     ----------
-    eq : Equilibrium
-        Equilibrium that will be optimized to satisfy the Objective.
+    eq : Equilibrium or FourierRZToroidalSurface
```

Comment:
Should we generalize the name to `thing`? In theory this should be able to be anything with other toroidal surface parameterizations.

Reply:
talked with @f0uriest and we wanna minimize API breaking changes